packages/engine-test-utils/src/__tests__/common-all/dnode.spec.ts

```diff
@@ -95,3 +97,74 @@ describe(`NoteUtils tests:`, () => {
     });
   });
 });
+
+describe(`SchemaUtil tests:`, () => {
```

Comment:
heads up for the tests: we follow a bdd style that is outlined here -> [[Style|dendron://dendron.docs/dev.process.qa.style]]